node/core/dispute-coordinator/Cargo.toml

```diff
@@ -17,6 +17,7 @@ polkadot-primitives = { path = "../../../primitives" }
 polkadot-node-primitives = { path = "../../primitives" }
 polkadot-node-subsystem = { path = "../../subsystem" }
 polkadot-node-subsystem-util = { path = "../../subsystem-util" }
+polkadot-node-core-approval-voting = { path = "../approval-voting" }
```

Comment:
We usually avoid depending on subsystems from one another. I'd recommend putting the constant into `polkadot-node-primitives` and having all of them draw it out from there.